gma/integration_test/src/integration_test.cc

```diff
@@ -349,6 +352,8 @@ void FirebaseGmaPreInitializationTests::SetUpTestSuite() {
 #endif  // defined(ANDROID)
 }
 
+#ifndef DEBUG_ONLY_TEST_UMP
```

Comment:
Is this meant to still be here?  If so, a comment about it would be good.

Reply:
Removing.